README.md

```diff
@@ -1,13 +1,56 @@
-# Northcoders News API
+# Northcoders News - Backend
 
-To setup your environment to run this project you will first need to create two _.env_ files to define the databases to use for testing or development. An example file, _.env-example_, has been provided.
+**An example backend for a social news site.**
 
-1. Duplicate this file as _.env.test_ and also _.env.development_ in the root directory.
-2. Edit both files to provide the database names you wish to use.
-3. Run
+A _'production'_ version is hosted at [nc-news-gjzo.onrender.com](https://nc-news-gjzo.onrender.com/api). When this example server is not in use it is spun down, so it may take up to a minute for the first request to respond.
 
+I built this as a portfolio project while on the _[Northcoders](https://northcoders.com) JavaScript full-stack course_. A seeded database was provided _(commit 9d279)_, after which all code is my own.
 
+- The runtime environment is **Node.js**
+- Testing uses the **Jest** and **Supertest** frameworks
+- Server functions use **Express**
+- Database functions use **PostgreSQL**
 
---- 
+A list of endpoints is available from [/api](https://nc-news-gjzo.onrender.com/api)
 
-This portfolio project was created as part of a Digital Skills Bootcamp in Software Engineering provided by [Northcoders](https://northcoders.com/)
+## How to run and test locally
+
+Requires [PostgreSQL](https://www.postgresql.org/download/) _v14.12+_ and [Node.js](https://nodejs.org/en/download/package-manager) _v22.3.0+_
+
+1. Clone the repo from [github.com/stevelw/be-nc-news](https://github.com/stevelw/be-nc-news)
+
+    `git clone https://github.com/stevelw/be-nc-news`
+
+2. Setup the environment
+
+    Create three _.env_ files in the root directory (only `.env.test` is needed to run the tests):
+    
+    1. `.env.test` with `PGDATABASE=nc_news_test`
+    2. `.env.development` with `PGDATABASE=nc_news`
```

Comment:
Would be better if you hide the actual database name in the readme for safety reason